pipeline-utils.groovy

```diff
@@ -139,6 +139,21 @@ def sh_capture(cmd) {
     return sh(returnStdout: true, script: cmd).trim()
 }
 
+def registry_login(username, password, registry) {
+    sh "podman login -u '${username}' -p '${password}' ${registry}"
+}
+
+// re-implementation of some functionality from scripts/pull-mount-oscontainer
+// sets up local container storage so the pipeline can pull container images
+// from inside another container
+def prep_container_storage(hostStorage, localStorage) {
```

Comment:
Hmm...maybe I chose the wrong names here...maybe they should be `oldStorage` and `newStorage`?

Reply:
How about using `hostStorage` and `containerStorage`? That may be a bit clearer.

Reply:
I think `hostStorage` was a mistake on my part.  

As far as I can tell, the host container storage isn't ever mounted into the assembler container (as of now), so we are really just deleting the "typical" location of the container storage in the container.  

For example,`scripts/pull-mount-oscontainer` deletes `/var/lib/containers` in its container, then bind mounts something like `$WORKSPACE/containers` to `/var/lib/containers`.  https://github.com/openshift/os/blob/master/scripts/pull-mount-oscontainer#L21-L23)

I think something more accurate would be `oldContainerStorage` and `newContainerStorage`

What do you think?

Reply:
I actually discovered that podman/buildah support a `--root` argument and have been using that in https://github.com/coreos/coreos-assembler/pull/90